prowler/providers/aws/lib/audit_info/audit_info.py

```diff
@@ -21,6 +21,7 @@
         session_duration=None,
         external_id=None,
     ),
+    audit_resources=None,
```

Comment:
I prefer to give this variable a more expressive name like `filtered_audit_resources` or `included_audit_resources`